ansible/roles/jenkins-worker/templates/centos.initd.j2

```diff
@@ -17,11 +17,12 @@ PIDFILE=/var/run/jenkins/$NAME.pid
 JENKINS_SLAVE_USER="iojs"
 JENKINS_SLAVE_JAR="/home/{{ server_user }}/slave.jar"
 JENKINS_SLAVE_LOG="/home/{{ server_user }}/$NAME.log"
-JENKINS_SLAVE_ARGS="-Xmx{{ server_ram|default('128m') }} -jnlpUrl {{ jenkins_url }}/computer/{{ inventory_hostname }}/slave-agent.jnlp -secret {{ secret }}"
+JENKINS_JAVA_ARGS="-Xmx{{ server_ram|default('128m') }}"
+JENKINS_SLAVE_ARGS="-jnlpUrl {{ jenkins_url }}/computer/{{ inventory_hostname }}/slave-agent.jnlp -secret {{ secret }}"
 JENKINS_ENV="JOBS={{ ansible_processor_vcpus }} \
   HOME=/home/{{ server_user }} \
-  DESTCPU={{ ansible_architecture }} \
-  ARCH={{ ansible_architecture }} \
+  DESTCPU={{ server_arch | default(ansible_architecture) }} \
+  ARCH={{ server_arch | default(ansible_architecture) }} \
```

Comment:
centos will do `x86_64` which our config scripts don't recognise so we have to override, I think maybe we should fix configure to recognise it as an alias

Reply:
Does this need to be manually defined every time you run? If so, and if it defaults to `x86_64`, then the chance of us missing this seems high.